ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/AdminResponseHandlers.java

```diff
@@ -35,10 +35,119 @@ private AdminResponseHandlers() {
   static void handleListStreamsResponse(
       final JsonObject streamsListEntity,
       final CompletableFuture<List<StreamInfo>> cf
+  ) {
+    try {
+      cf.complete(getListStreamsResponse(streamsListEntity).get());
```

Comment:
Just a sanity check probably best to check if the optional value is present before calling get(). (Same with below too)

Reply:
Do you mean switching the try-catch to an if-present-else-throw check? That makes sense to me. I'll make the change.